src/Cms/Email.php

```diff
@@ -123,7 +122,7 @@ protected function template(): void
 					$this->props['body']['text'] = $text->render($data);
 				}
 
-				// fallback to single email text template
+			// fallback to single email text template
```

Comment:
Same here

Reply:
I think this one makes actually sense

Reply:
Ah, right. Missed that this one was changed in the opposite direction. 👍